main.go

```diff
@@ -99,10 +101,11 @@ func main() {
 	le := utils.GetLeaderElectionConfig(restConfig, enableLeaderElection)
 
 	mgr, err := ctrl.NewManager(restConfig, ctrl.Options{
-		Namespace:              namespace,
+		Cache: cache.Options{
+			DefaultNamespaces: map[string]cache.Config{namespace: {}},
+		},
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
This doesn't need to be set, the new API defaults to 9443 already.
https://github.com/openshift/local-storage-operator/blob/e2eec8b9380c62a2eebeb0b893ea59350e3f6914/vendor/sigs.k8s.io/controller-runtime/pkg/webhook/server.go#L76-L78